index.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const bcrypt = require('bcryptjs');
+const db = require('./data/db.js');
+const helmet = require('helmet');
+
+const server = express();
+
+server.use(express.json());
+server.use(helmet());
+
+server.get('/', (req, res) => {
+  res.send({ message: 'I am alive' });
+});
+
+//Register 
+server.post('/api/register', (req, res) => {
+  const creds = req.body;
+  const hash = bcrypt.hashSync(creds.password, 14);
+  creds.password = hash;
+  db('users').insert(creds).then(id => res.status(201).json(id)).catch(err => res.json(err))
+})
+
+//Login
+server.post('/api/login', (req, res) => {
+  const creds = req.body;
+  db('users').where({ username: creds.username }).first().then(user => {
```

Comment:
I recommend making your endpoints for readable.  As they are, they seem too crammed together.  When using .where or .first or anything as such, try dropping them down a line to help with spacing.